### DISCUSSION TITLE
Charts: Allow for callbacks with tooltip and/or export ChartTooltip in a way that it can be wrapped

### DISCUSSION THREAD

**Discussion by aeftimia** [mantinedev/mantine#6865] [Feature requests]
I love Mantine's Charts package and I'd like to build some custom functionality that absorbs information gathered from the tooltip to draw on the plot. In particular, I'd like to do this for the AreaChart. Unfortunately, the default tooltip content, ChartTooltip, takes in information passed from hooks executed within AreaChart, so I can't easily import it, wrap it, and pass it back to tooltipProps.content to store the information it gets elsewhere.

**Comment by rtivital** [comment]
You can pass any data to the tooltip component with custom tooltip – https://mantine.dev/charts/area-chart/#custom-tooltip

**Reply by aeftimia** [reply]
Wow I honestly missed that example. When I swap out the example tooltip content for the `ChartTooltip` provided within Mantine/Charts, it does replicate the default tooltip exactly even without the `classNames` and `styles` props!
